packages/story-editor/src/components/library/panes/media/local/hotlink/useInsert.js

```diff
@@ -77,18 +79,22 @@ function useInsert({ link, setLink, setErrorMsg, onClose }) {
       } = hotlinkData;
 
       try {
-        const resource = await getResourceFromUrl(link, type);
+        const proxiedUrl = getProxiedUrl({ isExternal: true }, link);
+        const resource = await getResourceFromUrl(proxiedUrl, type);
+        resource.alt = originalFileName;
+        resource.src = link;
```

Comment:
https://github.com/google/web-stories-wp/pull/9209/files#r719748366

Oh, is that why?

Reply:
We need to pass the proxied version of the link into the `getResourceFromUrl`, as this generated posters, muted and image sizes. These actions need a function that can access the resource. So we pass in the proxied version and reset it back.